js/arena.js

```diff
@@ -0,0 +1,51 @@
+import {
+    Gladiator
+} from './gladiator.js';
+class Arena extends Gladiator {
+    constructor(name) {
+        super();
+        this.name = name;
```

Comment:
The name needs to capitalized, meaning the first letter is upper case, while the rest are lower case